ccf-dynamodb/cloudformation/dynamodb.yml

```diff
@@ -37,10 +37,10 @@ Resources:
     Properties: 
       TableName: !Sub ccf-client-settings-${EnvironmentName}
       AttributeDefinitions: 
-        - AttributeName: "client_id"
+        - AttributeName: "licence_id"
           AttributeType: "S"
       KeySchema: 
-        - AttributeName: "client_id"
+        - AttributeName: "licence_id"
```

Comment:
As far as I understand this table will hold the feed ID too, so in order to not make another DB update when this is introduced I suggest this to be a composite primary key with the feed ID as a sort key.

Reply:
On a second thought, this would require changes in the RW app, so if you think that there won't be a problem to update the DB again, then maybe it's better to keep the PR as it is.

Reply:
A third thought 😄  After further reading the other PR, it looks that no changes are needed there.

Reply:
I will add the changes for the feed ID as part of the next task because I am not sure if this will be the only change related to feeds. There should be a simple code change to provide non-empty value for the feed ID [in `CreateLicence` method](https://github.com/Financial-Times/ccf-client-settings-api/blob/feature/UPPSF-1645-licence-endpoints/dynamodb.go#L35) because it is not possible to have empty values for fields that are part of a key or an index.